environment-dev.yml

```diff
@@ -2,7 +2,7 @@ name: gmso-dev
 channels:
   - conda-forge
 dependencies:
-  - python>=3.8
+  - python<3.12
```

Comment:
We probably should also include greater than 3.8 right?

Reply:
yes, it should